wa/workloads/jetnews/__init__.py

```diff
@@ -0,0 +1,48 @@
+#    Copyright 2023 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from wa import Workload
```

Comment:
Thanks for the review. Given the fact I haven't actually implemented this code (someone else did), I'd say it might be unfamiliarity with how WA works and its modules. I'll have to touch these up and make them do the right thing. So I might have a few questions here and there, as I get familiar with this code.